hello.apex

```diff
@@ -1,10 +1,17 @@
-String flagKey = 'my-flag';
+# You will set your LaunchDarkly SDK key during the bridge.
+
+# Set flagKey to the feature flag key you want to evaluate.
+String flagKey = 'my-boolean-flag';
 
 LDConfig config = new LDConfig.Builder().build();
 
 LDClient client = new LDClient(config);
 
-LDUser user = new LDUser.Builder('abc').build();
+# Set up the user properties. This user should appear on your LaunchDarkly users
+# dashboard soon after you run the demo.
+LDUser user = new LDUser.Builder('example-user-key')
+    .setName('Sandy')
+    .build();
 
 if (client.boolVariation(user, flagKey, false)) {
     System.debug('feature is on');
```

Comment:
Can you update these so that the output message matches the spec?

You could use two versions (line 10 and 12), "Feature flag is true for this user", "Feature flag is false for this user". You could also combine, it'd be something like:
```
# replacing lines 9-13

# taken from https://docs.launchdarkly.com/sdk/features/evaluating#apex
Boolean value = client.boolVariation(user, flagKey, false); 

# not 100% sure on syntax here
System.debug('Feature flag ', flagKey, ' is ', value, ' for this user');
```

Reply:
I couldn't quite figure out how it would work to combine the two, so left them separate for now.

Reply:
Believe it should be:
```apex
System.debug('Feature flag '+flagKey+' is '+value+' for this user');
```

Reply:
Thank you!